pkg/controller/toolchainstatus/toolchainstatus_controller_test.go

```diff
@@ -74,6 +75,34 @@ func prepareReconcile(t *testing.T, requestName string, httpTestClient *fakeHTTP
 	return r, reconcile.Request{NamespacedName: test.NamespacedName(test.HostOperatorNs, requestName)}, fakeClient
 }
 
+func prepareReconcileWithStatusConditions(t *testing.T, requestName string, httpTestClient *fakeHTTPClient,
+	getMemberClustersFunc func(fakeClient client.Client) cluster.GetMemberClustersFunc, conditions []toolchainv1alpha1.Condition, initObjs ...runtime.Object) (*ReconcileToolchainStatus, reconcile.Request, *test.FakeClient) {
```

Comment:
the `httpTestClient` as well as `getMemberClustersFunc` is not used in this function

Reply:
Right. Cleaned it up in https://github.com/codeready-toolchain/host-operator/pull/332/commits/ebca00c3f55356cdfcda58e66fa42f542df28302